2-mandatory/5-writing-tests.js

```diff
@@ -35,41 +35,57 @@ function convertScoreToGrade(score) {
   passes.
 */
```

Comment:
It looks like there's a problem with the way these tests are written. If you run `npm test -- --testPathPattern 5-writing-tests.js` from the terminal at the root of your project - you'll see this message: "Expect takes at most one argument."
Any ideas how you can fix this? Maybe have a look at some of the other files for an example of how to use the `expect` function.